Automattic/wp-calypso
# Migrate `blogStickers` to use ReactQuery

#### Changes proposed in this Pull Request

Motivated by #63350 I took the opportunity to migrate blog stickers to React Query and clean up any redux related code.

* Migrate blog stickers to use ReactQuery
  * Create and use `useBlogStickersQuery`
  * Create and use `use(Add|Remove)BlogStickersMutation`
  * Remove any Redux and data layer related code

#### Testing instructions

* Follow the testing instructions from #63350, they should still work exactly as described
* Now choose a blog you own which has at least 1 post (you should be on `/read/feeds/:siteId`)
* Click on the three-dots menu for a post and apply the `a8c-test-blog` sticker, then remove it again after you received a success message. You can also test the error case by blocking requests to `public-api.wordpress.com/rest/v1.1/sites/*/blog-stickers`.
